<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DFW2LPTSFH9" name="Default test" tests="11" failures="1" timestamp="19 Feb 2016 21:03:46 GMT" time="41.149" errors="0">
  <testcase name="NavigateToPartyPage" time="0.223" classname="com.norwex.nco.A_Party"/>
  <testcase name="createParty" time="19.982" classname="com.norwex.nco.A_Party">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.norwex.nco.A_Party.createParty(A_Party.java:60)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- createParty -->
  <testcase name="ConfirmShipping" time="0.0" classname="com.norwex.nco.A_Party">
    <skipped/>
  </testcase> <!-- ConfirmShipping -->
  <testcase name="addGuests" time="0.0" classname="com.norwex.nco.A_Party">
    <skipped/>
  </testcase> <!-- addGuests -->
  <testcase name="addGuestOrders" time="0.0" classname="com.norwex.nco.A_Party">
    <skipped/>
  </testcase> <!-- addGuestOrders -->
  <testcase name="UpdateGuestOrders" time="0.0" classname="com.norwex.nco.A_Party">
    <skipped/>
  </testcase> <!-- UpdateGuestOrders -->
  <testcase name="addHostOrder" time="0.0" classname="com.norwex.nco.A_Party">
    <skipped/>
  </testcase> <!-- addHostOrder -->
  <testcase name="ReviewBookings" time="0.0" classname="com.norwex.nco.A_Party">
    <skipped/>
  </testcase> <!-- ReviewBookings -->
  <testcase name="FinalPayment" time="0.0" classname="com.norwex.nco.A_Party">
    <skipped/>
  </testcase> <!-- FinalPayment -->
  <testcase name="HostPlanSummary" time="0.0" classname="com.norwex.nco.A_Party">
    <skipped/>
  </testcase> <!-- HostPlanSummary -->
  <testcase name="VerifySummary" time="0.0" classname="com.norwex.nco.A_Party">
    <skipped/>
  </testcase> <!-- VerifySummary -->
</testsuite> <!-- Default test -->
